lib/open_project/github_integration/notification_handlers.rb

```diff
@@ -35,8 +35,10 @@ module NotificationHandlers
     #   repository: <the repository in action>
     # }
     def self.pull_request(payload)
-      # Don't add comments on new pushes to the pull request
-      return if payload['action'] == 'synchronize'
+      # Don't add comments on new pushes to the pull request => ignore synchronize.
+      # Don't add comments about assignments and labels either.
+      ignored_actions = %w['synchronize' 'assigned' 'unassigned' 'labeled' 'unlabeled']
+      return if ignored_actions.include? payload['action']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.